docs/content/configuration/broker.md

```diff
@@ -55,6 +55,7 @@ The broker uses processing configs for nested groupBy queries. And, optionally,
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.processing.buffer.sizeBytes`|This specifies a buffer size for the storage of intermediate results. The computation engine in both the Historical and Realtime nodes will use a scratch buffer of this size to do all of their intermediate computations off-heap. Larger values allow for more aggregations in a single pass over the data while smaller values can require more passes depending on the query that is being executed.|1073741824 (1GB)|
+|`druid.processing.buffer.poolCacheMaxCount`|processing buffer pool caches the buffers for later use, this is the maximum count cache will grow to.|Integer.MAX_VALUE|
```

Comment:
That's not quite true. If I have a max size of 10 and I request 100 buffers, I'll still get 100 buffers, but  I'll only return 10 for later use. Then if I request 100 I'll have 10 queued and 90 new ones. And when they are done they will return 10 and drop 90 to GC.